### PR TITLE
export util

### LINKED ISSUE #34: export util functions directly
I recommend exporting these functions directly so that they can be tree shaken, etc. e.g. this file would export `_smolr` etc rather than `utils.*`. Think it's the same amount or less code.

_Originally posted by @psugihara in https://github.com/okwme/anybody-problem/pull/30#discussion_r1516663488_

### PR DESCRIPTION
**Pull request by okwme**
closes #34 

 seems like everything still works but i realize something is going on…in some of the tests that shoudl be more fully addressed in a separate issue

### CODE REVIEW

**Comment by okwme** [comment]
will this satisfy the tree shaking thing you mentioned @psugihara ?

File: src/anybody.js [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
+  _calculateTime,
+  _explosion,
+  _addVectors,
   _validateSeed
-} = utils
-
-export { utils }
-
-
+}
```
Comment (psugihara) [inline src/anybody.js:31]: Re-exporting here makes it so all of these will be included in the bundle if anybody is included (whether or not they are all used). Instead I'd suggest removing this export and just importing what you need where you need it directly from `calculations.js`.

**Review by psugihara — COMMENTED** [review]
almost

**Comment by psugihara** [comment]
"You can imagine your application as a tree. The source code and libraries you actually use represent the green, living leaves of the tree. Dead code represents the brown, dead leaves of the tree that are consumed by autumn. In order to get rid of the dead leaves, you have to shake the tree, causing them to fall."

🍃

https://webpack.js.org/guides/tree-shaking/#

**Comment by okwme** [comment]
how bout now? @psugihara

**Pull request merged**